yaml/boundaries.yaml

```diff
@@ -327,6 +317,7 @@ filters:
       <<: [*output_properties, *ne_localized_kind_properties]
       kind: unrecognized_macroregion
       kind_detail: '4'
+      dispute_id: { col: 'ne:brk_a3' }
```

Comment:
Repeating... To indicate both the dispute grouping and the unique line segments, concat with `ne_id` with a `_` spacer. The result would be something like `B05_123456789`

Reply:
Switch to using global, please.

Reply:
Fixed in https://github.com/tilezen/vector-datasource/pull/2054/commits/6f75fc78a61e32afd27c6a36481a8d2ba9386bdf